firmware-binaries/test-cases/clock-control-wb/src/main.rs

```diff
@@ -7,11 +7,9 @@
 
 use core::panic::PanicInfo;
 
-use bittide_sys::{
-    clock_control::{ClockControl, SpeedChange},
-    debug_register::DebugRegister,
-    uart::Uart,
-};
+use bittide_hal::shared::devices::clock_control::{ClockControl, SpeedChange};
+use bittide_hal::shared::devices::debug_register::DebugRegister;
+use bittide_sys::uart::Uart;
```

Comment:
Why not use the `bittide_hal::shared::devices::uart::Uart`? Less work for when we see what to keep from `bittide_sys` and what not (I guess the UART wouldn't be part of it though, so it would be (re)moved anyway)

Reply:
Oh you're saying UART is already automatically generated and I can boot out another file?

Reply:
I'll create an issue out of it. I'd like to get some help from you/Ryan to see how we can best do this :)